src/azure-cli/azure/cli/command_modules/network/tests/latest/zone_files/zone10.txt

```diff
@@ -7,13 +7,14 @@ $ORIGIN zone10.com.
               300 ; retry
               2419200 ; expire
               300 ; minimum
-              )
+              );
 
   172800 IN NS ns1-03.azure-dns.com.
   172800 IN NS ns2-03.azure-dns.net.
   172800 IN NS ns3-03.azure-dns.org.
   172800 IN NS ns4-03.azure-dns.info.
-;
+
+@ NS ns1-03.azure-dns.com.
```

Comment:
why is this needed?

Reply:
Accidentally changed it. Reverted back to original file.